superset/migrations/versions/8d49a37823bf_make_table_unique_within_db_and_schema.py

```diff
@@ -1,3 +1,19 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
another PR bleeding through?

Reply:
ran `rodent apply` and figured might as well add missing headers, I could split the PR but I don't think it's worth it.

Reply:
It is if we cut the next build right after this PR. :)